cpp/examples/strings/custom_optimized.cu

```diff
@@ -37,9 +37,9 @@
  * @param d_visibilities Column of visibilities
  * @param d_sizes Output sizes for each row
  */
-__global__ void sizes_kernel(cudf::column_device_view const d_names,
-                             cudf::column_device_view const d_visibilities,
-                             cudf::size_type* d_sizes)
+__global__ static void sizes_kernel(cudf::column_device_view const d_names,
```

Comment:
So should these instances be replaced by `CUDF_KERNEL`? (Same for below).

Reply:
These are example code so we want to demonstrate the appropriate pattern, but not rely on cuDF internal macros.

Reply:
@bdice That was my exact reasoning as well.